README.md

```diff
@@ -279,7 +279,7 @@ The following example scans the outfile log in file QTEMP/STDOUTQSH for a file n
 
 **SCANFOR** - Text value to scan for on each line in the stdout outfile. The value passed IS case sensitive and must match the value in the log file.
 
-**EXACTMATCH** - *YES - Value must match exactly and be the only thing on the selected line. *NO - At least one line must contain the value somewhere in the line. Good for generic matching.
+**EXACTMATCH** - *YES - Value must match exactly and be the onLy thing on the selected line. *NO - At least one line must contain the value somewhere in the line. Good for generic matching.
 
 # QSHPATH command parms
```

Comment:
Was there a reason you uppercases the L on only ?  I don't see any other changes ??

Reply:
The "L" already looked uppercase, so I was trying to lowercase it. But maybe it was already lowercase but the typeface made it look uppercase. Looks identical. I didn't mean to change it, really.

![image](https://user-images.githubusercontent.com/5905791/118157579-b8d9d400-b3e8-11eb-86e9-85c13b9b57aa.png)